lib/LaTeXML/resources/XSLT/LaTeXML-webpage-xhtml.xsl

```diff
@@ -548,22 +550,24 @@
           <xsl:attribute name="style">letter-spacing:-0.2em; margin-right:0.1em;</xsl:attribute>
           <xsl:text>L</xsl:text>
           <xsl:element name="span" namespace="{$html_ns}">
-            <xsl:attribute name="style">font-size:70%;position:relative; bottom:2.2pt;</xsl:attribute>
-            <xsl:text>A</xsl:text>
+            <xsl:attribute name="class">ltx_font_smallcaps</xsl:attribute>
+            <xsl:attribute name="style">position:relative; bottom:2.2pt;</xsl:attribute>
```

Comment:
I like the change to relative units - the `2.2pt` here now seems to be the exception to the rule. 

Flagging it in case you would have liked to switch it.